vcnet/regex/debug.c

```diff
@@ -15,14 +15,12 @@
  == void regprint(regex_t *r, FILE *d);
  */
 void
-regprint(r, d)
-regex_t *r;
-FILE *d;
+regprint(regex_t *r, FILE *d)
```

Comment:
The regex code in the vcnet directory is not maintained by the CUPS project. No changed will be accepted here.